src/index.d.ts

```diff
@@ -87,7 +87,7 @@ export type ComponentProps<
 	: never;
 
 export interface FunctionComponent<P = {}> {
-	(props: RenderableProps<P>, context?: any): ComponentChild;
+	(props: RenderableProps<P>, context?: any): VNode<any>;
```

Comment:
This didn't revert return type to `VNode<any> | null`, as it was in v10.10.3: https://github.com/preactjs/preact/pull/3676/files#diff-619cc2239a7eb4ef05041767e6ab0e4fb069e98ee9d2c83c2d8e1865c475e141L90

Returning `null` from components is valid, as described here:
React Docs > [Preventing Component from Rendering](https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering)